app/models/crop.rb

```diff
@@ -33,6 +33,47 @@ class Crop < ActiveRecord::Base
       :with => /^https?:\/\/en\.wikipedia\.org\/wiki/,
       :message => 'is not a valid English Wikipedia URL'
     }
+  
+  include Elasticsearch::Model
+  include Elasticsearch::Model::Callbacks
+  index_name [Rails.env, "growstuff"].join('_')
+  settings index: { number_of_shards: 1},
+    analysis: {
+      tokenizer: {
+        gs_edgeNGram_tokenizer: {
+          type: "edgeNGram",
+          min_gram: 2,
+          max_gram: 20,
+          token_chars: [ "letter", "digit" ]
+        }
+      },
+      analyzer: {
+        gs_edgeNGram_analyzer: {
+          tokenizer: "gs_edgeNGram_tokenizer",
+          filter: ["lowercase"]
+        }
+      },
+    } do
+    mappings dynamic: 'false' do
+      indexes :id, type: 'long'
```

Comment:
@Skud That's a good point. Actually, now that I think about it, there really isn't a use case, is there? I think I had plantings and crops mixed up in my brain. So yeah, no need to index more than is already here. Plus, I guess the number of crops will only ever be in the thousands, so it's easy to re-index the whole thing if we wanted to search on more criteria.

One thing just occured to me, @oshiho3 : When a crop is edited or